plugin.video.mytv_bg/addon.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#imports
+from xbmcswift import Plugin, xbmc, xbmcaddon, xbmcplugin, xbmcgui, clean_dict
+from urllib2 import urlopen, HTTPError
+import xbmcgui
+from pyxbmct.addonwindow import *
+import sys
+import json
+import os.path
+from urllib2 import Request
+import urllib
+import urllib2
+
+__addon_name__		= xbmcaddon.Addon().getAddonInfo('name')
+__id__				= xbmcaddon.Addon().getAddonInfo('id')
+__lang__			= xbmcaddon.Addon().getLocalizedString
+__addon__ 			= xbmcaddon.Addon()
```

Comment:
if you define `__addon__` , then you can probably just use the same instance to get the id, name and all the other stuff instead of instantiating new objects. Just move the  `__addon__` to top and reuse it. E.g:
`__addon__` = xbmcaddon.Addon()`
`__addon_name__ = __addon__,getAddonInfo('name')`